More arousal logging; adds back aphro/nympho

Putnam3145 opened the pull request:
## About The Pull Request

I was a goddamn idiot to just remove instead of adding better logging. Now all sources of arousal as well as climaxes are logged in the emote log, including whether they're forced or not.

## Why It's Good For The Game

Because the logging is good enough, we can bring back stuff like this without too much issue.

## Changelog
:cl:
add: (Hexa)crocin
add: (Hexa)camphor
add: Nymphomaniac quirk
admin: All climaxes and arousals are now logged, as well as genital exposure.
/:cl:

Putnam3145 committed:
Adds back aphro, nympho

Putnam3145 committed:
Slightly more logging, whoops added pref

Putnam3145 committed:
should also log climaxes (all of them)

Putnam3145 committed:
added exposure logging

LetterN commented on code/modules/mob/living/carbon/human/species.dm:
```diff
@@ -1547,9 +1547,9 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
+		target.adjust_arousal(20,"masochism", maso = TRUE)
 		if (ishuman(target) && HAS_TRAIT(target, TRAIT_MASO) && target.has_dna() && prob(10))
-			target.mob_climax(forced_climax=TRUE)
+			target.mob_climax(forced_climax=TRUE, "masochism")
```
```suggestion
			target.mob_climax(forced_climax=TRUE, cause = "masochism")
```

LetterN commented on code/modules/mob/living/carbon/human/species.dm:
```diff
@@ -1931,7 +1931,7 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 					H.update_damage_overlays()
 					if(HAS_TRAIT(H, TRAIT_MASO) && prob(damage_amount))
-						H.mob_climax(forced_climax=TRUE)
+						H.mob_climax(forced_climax=TRUE, "masochism")
```
```suggestion
						H.mob_climax(forced_climax=TRUE, cause  = "masochism")
```

LetterN commented on code/modules/vending/kinkmate.dm:
```diff
@@ -31,6 +31,8 @@
 				/obj/item/storage/pill_bottle/penis_enlargement = 5,
 				/obj/item/storage/pill_bottle/breast_enlargement = 5,
+				/obj/item/reagent_containers/glass/bottle/crocin = 5,
+				/obj/item/reagent_containers/glass/bottle/camphor = 5
```
```suggestion
				/obj/item/reagent_containers/glass/bottle/camphor = 5,
```

LetterN commented on modular_citadel/code/datums/status_effects/chems.dm:
```diff
@@ -500,13 +500,19 @@
+			else if (lewd && lowertext(customTriggers[trigger]) == "cum")//aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+				if(ishuman(C))
+					var/mob/living/carbon/human/H = C
+					H.mob_climax(forced_climax=TRUE, "MKUltra")
```
```suggestion
					H.mob_climax(forced_climax=TRUE, cause = "MKUltra")
```

Putnam3145 committed:
keyword argument funnies, whoops comma

Putnam3145 committed:
Fixed a runtime might cause issues.

This might also break logging completely, which, why. This isn't a high-priority log, though, so.

Putnam3145 committed:
undergarment logging too

Putnam3145 committed:
don't push before thinking, kids
